ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/metering_report_service.proto

```diff
@@ -50,6 +50,10 @@ message GetMeteringReportResponse {
 
     // The time at which the report was computed.
     google.protobuf.Timestamp report_generation_time = 3;
+
+    // A hash that can be used to verify the consistency of the report
+    string check = 4;
```

Comment:
As explained on the corresponding design doc, let's see whether we can add a bit more structure here so that:
1. We can reuse an existing standard for authenticating JSON data like JWS https://datatracker.ietf.org/doc/html/rfc7515
2. We have enough structure for the authentication data to rotate keys and add or remove fields from the report.